edx_repo_tools/codemods/django3/github_actions_modernizer_django.py

```diff
@@ -0,0 +1,72 @@
+"""
+Django Matrix Modernizer for Github Actions CI
+"""
+import re
+from copy import deepcopy
+
+import click
+
+from edx_repo_tools.utils import YamlLoader
+
+DJANGO_PATTERN = r"django[0-3][0-2][0-2]?"
+DJANGO_VERSIONS = ['2.2', '3.0', '3.1', '3.2']
+ALLOWED_DJANGO_ENVS = ['django22', 'django30', 'django31', 'django32']
+
+
+class GithubCIDjangoModernizer(YamlLoader):
+    def __init__(self, file_path):
+        super().__init__(file_path)
+        self.yml_instance.default_flow_style = None
+        self.yml_instance.indent(mapping=2, sequence=2, offset=0)
+
+    def _update_matrix(self):
+
+        for section_key in self.elements['jobs'].keys():
+            matrix_keys = self.elements['jobs'][section_key]['strategy']['matrix'].keys()
```

Comment:
in case we've multiple jobs defined in a workflow, this will only keep the matrix value for the last job specified

Reply:
In case of [multiple jobs](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_id), we will get the CI structure like `['jobs'][{first job}, {second job}, ...]` and the current implementation will update all of the jobs in the list since it is running a loop for all the jobs.